src/VisualStudio/Xaml/Impl/Implementation/IXamlEditAndContinueSolutionProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
something bad with teh merges from main.  these will disappear once the branches sync up.